cmd/state/autoupdate.go

```diff
@@ -84,12 +84,19 @@ func autoUpdate(args []string, cfg *config.Instance, out output.Outputer) (bool,
 			logging.Debug("Update already in progress")
 			return false, nil
 		}
+		fmt.Println("err:", err)
 		return false, locale.WrapError(err, "auto_update_failed")
 	}
 
 	out.Notice(locale.Tr("auto_update_relaunch"))
 	out.Notice("") // Ensure output doesn't stick to our messaging
-	code := relaunch(args)
+
+	stateInfo, err := installation.NewAppInfo(installation.StateApp)
+	if err != nil {
+		return false, locale.WrapError(err, "err_state_info")
+	}
+
+	code := relaunch(stateInfo.Exec(), args)
```

Comment:
This feels like it's logic for inside of `relaunch`, not something we pass to it.